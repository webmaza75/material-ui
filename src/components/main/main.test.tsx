import * as React from 'react';
import * as ShallowRenderer from 'react-test-renderer/shallow';

import Main from './main';

describe(`Main`, () => {
  it(`Main renders correctly`, () => {
    const renderer = ShallowRenderer.createRenderer();
    const result = renderer
      .render(<Main />);

    expect(result).toMatchSnapshot();
  });
});