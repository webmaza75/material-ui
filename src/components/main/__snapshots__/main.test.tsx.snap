// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Main renders correctly 1`] = `
<div
  style={
    Object {
      "flexGrow": 1,
    }
  }
>
  <Navbar />
  <AppPanel />
  <div
    style={
      Object {
        "marginBottom": 20,
        "marginLeft": "auto",
        "marginRight": "auto",
        "marginTop": 150,
        "width": "80%",
      }
    }
  >
    <AppTable
      onItemSelect={[Function]}
      selected={Array []}
    />
  </div>
</div>
`;
